stdlib/LinearAlgebra/src/schur.jl

```diff
@@ -190,6 +190,10 @@ function schur(A::Bidiagonal{T}) where {T}
         return Schur(Matrix{t}(Bidiagonal(dv, ev, 'U')), J, dv)
     end
 end
+function schur(A::AbstractMatrix{TA}, B::AbstractMatrix{TB}) where {TA,TB}
```

Comment:
This should be moved to line 357 or so, as it is related to the generalized Schur-decomposition.

Reply:
Gotcha.